hack/verify-godeps.sh

```diff
@@ -53,6 +29,9 @@ if ! [[ ${KUBE_FORCE_VERIFY_CHECKS:-} =~ ^[yY]$ ]] && \
   exit 0
 fi
 
+# Ensure we have the right godep version available
+kube::util::ensure_godep_version
```

Comment:
Why do we need this?  Both godep-restore.sh and godep-save.sh call this...

Reply:
The issue with that is that we create a new temporary gopath to restore/save the deps, and that GOPATH/bin won't ever be in the path, as it's in a randomly generated temp folder.

Reply:
> Why do we need this?

What is the issue with this call? Is it the "ensure" that you don't like? I.e. we could just check and complain, or install into a temporary dir.

Reply:
It's disconnected from where it is being used, and it is already called in teh places where it is being used (or should be).

Reply:
@cblecker - not sure I understand your point.  Wherever we change GOPATH we should also change PATH